access-token-management/src/AccessTokenManagement.OpenIdConnect/OpenIdConnectTokenManagementServiceCollectionExtensions.cs

```diff
@@ -130,6 +150,28 @@ public static IHttpClientBuilder AddUserAccessTokenHttpClient(this IServiceColle
             .AddUserAccessTokenHandler(parameters);
     }
 
+    /// <summary>
+    /// Adds a named HTTP client for the factory that automatically sends the current user access token
```

Comment:
```suggestion
    /// Adds a typed HTTP client for the factory that automatically sends the current client access token. The client access token is an access token that is not associated with any user, obtained with the client credentials flow.
```

Reply:
The named version of `AddClientAccessTokenHttpClient` also has the second typo where it says it is sending the user's access token. Let's update that as well.